frontend/src/game/play/play.js

```diff
@@ -7,23 +7,27 @@ const config = require('config/config.js');
 const backend = config.servers["backend"];
 
 module.exports = {
-    playCard(rownum, cardindex, target) {
-        const playData = {
-            index: cardindex,
-            row: rownum,
-            target: target,
-        };
-        document.querySelector('.card[index="'+cardindex+'"]').style.background = "yellow";
+    playCard(target) {
```

Comment:
Since we keep track of the `status.clickedCard` for marking it and handling abilities, passing the playing info here is redundant.